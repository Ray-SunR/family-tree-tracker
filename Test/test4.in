! D
! C
! B
! A
+ A E
+ A F
! King
p
* B
p
+ B C
+ B D
? E
? D
* G
p
+ A B
+ B C
+ B D
? E
? D
p
