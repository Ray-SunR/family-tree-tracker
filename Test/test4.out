King is parent of A
A is parent of B,E,F
B is parent of C
C is parent of D
King is parent of A
A is parent of E,F
Failed
Failed
King,A,E
Not found
King is parent of A
A is parent of E,F
King,A,E
King,A,B,D
King is parent of A
A is parent of E,F,B
B is parent of C,D
